package.json

```diff
@@ -45,5 +46,8 @@
     "eslint-plugin-react-hooks": "4.6.0",
     "jest": "29.5.0",
     "ts-jest": "29.1.0"
+  },
+  "prettier": {
+    "trailingComma": "es5"
```

Comment:
What prettier config? We can put this in a separate prettierrc if we want.

This is necessary because the default has changed from `es5` to `all` in the new prettier. Alternatively we can go change the other files in this repo that are affected to the new comma rules.

Reply:
For reference:

https://prettier.io/docs/en/options.html#trailing-commas